pdc/apps/module/routers.py

```diff
@@ -1,5 +1,10 @@
-from . import views
+#
+# Copyright (c) 2018 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+from pdc.apps.module.views import ModuleViewSet
 from pdc.apps.utils.SortedRouter import router
 
 
-router.register(r'unreleasedvariants', views.UnreleasedVariantViewSet)
+router.register(r'module', ModuleViewSet, base_name='module')
```

Comment:
A bit of bikeshedding: all REST-like urls use plural, so this should as well probably.

Reply:
Good point. I'll change this.